git-cliff-core/src/repo.rs

```diff
@@ -239,42 +239,36 @@ mod test {
 		.to_string())
 	}
 
-	#[test]
-	fn git_log() -> Result<()> {
-		let repository = Repository::init(
+	fn get_repository() -> Result<Repository> {
+		Repository::init(
 			PathBuf::from(env!("CARGO_MANIFEST_DIR"))
 				.parent()
 				.expect("parent directory not found")
 				.to_path_buf(),
-		)?;
+		)
+	}
+
+	#[test]
+	fn get_latest_commit() -> Result<()> {
+		let repository = get_repository()?;
 		let commits = repository.commits(None, None, None)?;
 		let last_commit =
 			AppCommit::from(&commits.first().expect("no commits found").clone());
 		assert_eq!(get_last_commit_hash()?, last_commit.id);
-		if let Err(e) = last_commit.into_conventional() {
-			match e {
-				Error::ParseError(e) => {
-					eprintln!("\nthe last commit is not conventional\n");
-					assert_eq!(ErrorKind::InvalidFormat, e.kind())
-				}
-				_ => {
-					unreachable!()
-				}
-			}
-		}
```

Comment:
I removed this check. I'm sure that the ability to parse a commit to conventional is well tested elsewhere.